### PR TITLE
Change in AKO statefulset status condition message

### PR DESCRIPTION
**Pull request by monotosh-avi**
The type of the condition is now ako.vmware.com/ObjectDeletionInProgress, which signifies if AKO is deleting all Avi Objects. The value of status would be True only when the delete operation is in progress, else it would be set False. In case of an error like timeout, it would be set to unknown. 

- The status of this condition would be set to true after setting deleteConfig to true, before starting all object deletion.
- The reason would be set to ObjectDeletionStarted.
- The status would be set to false when all objects gets deleted.
- The reason would be set to ObjectDeletionDone.

For example:
Status condition before deletion:
  - lastTransitionTime: "2020-12-15T10:10:45Z"
    message: Started deleting objects
    reason: ObjectDeletionStarted
    status: "True"
    type: ako.vmware.com/ObjectDeletionInProgress

Status condition after deletion:
  - lastTransitionTime: "2020-12-15T10:10:48Z"
    message: Successfully deleted all objects
    reason: ObjectDeletionDone
    status: "False"
    type: ako.vmware.com/ObjectDeletionInProgress

### CODE REVIEW

**Review by sudswasavi — DISMISSED** [review]

**Comment by vijay-avi** [comment]
build ako

**Comment by vijay-avi** [comment]
build ako

File: internal/lib/constants.go [hunk 1]
```diff
@@ -99,11 +99,11 @@ const (
...
 	AviObjDeletionTime                         = 30 // Minutes
-	AKOConditionType                           = "akoStatus"
+	AKOConditionType                           = "ako.vmware.com/ObjectDeletionInProgress"
 	AKOStatefulSet                             = "ako"
-	ObjectDeletionStartStatus                  = "objDeletionStarted"
-	ObjectDeletionDoneStatus                   = "objDeletionDone"
-	ObjectDeletionTimeoutStatus                = "objDeletionTimeout"
+	ObjectDeletionStartStatus                  = "ObjectDeletionStarted"
+	ObjectDeletionDoneStatus                   = "ObjectDeletionDone"
+	ObjectDeletionTimeoutStatus                = "ObjectDeletionTimeout"
```
Comment (shubhamavi) [inline internal/lib/constants.go:106]: since ObjectDeletion is common in all enums, can we chuck it out to just be `Started/Done/Timedout`?

**Review by shubhamavi — COMMENTED** [review]
minor nit.

File: internal/lib/constants.go (hunk 1 above)
Comment (monotosh-avi) [reply internal/lib/constants.go:106]: Sounds fine to me. @sudswasavi ?

Comment (sudswasavi) [reply internal/lib/constants.go:106]: yeah just on that thought, I wonder how we signify what objects these are. In this case, there are LB Objects I think. Thoughts?

**Review by sudswasavi — APPROVED** [review]

**Review by shubhamavi — APPROVED** [review]

**Pull request merged**